lakerathe/Kanthurn.lua

```diff
@@ -0,0 +1,13 @@
+--Kanthurn (51096)
+-- Paladin 1.0 Epic Quest npc giving hints
+-- https://everquest.allakhazam.com/db/npc.html?id=24011
+--
+-- related to PAL 1.0. Verified on live the text is availabe for all classes.
+
+function event_say(e)
+	if(e.message:findi("hail")) then
+        e.self:Say("Hello, " .. e.other:GetName() .. ". Have you heard of [Rineval] Talyas?");
```

Comment:
indenting, don't be like @huffin please choose one type of indenting and utilize that for consistency. Other than that, looks good!

Reply:
Thanks! Fixed inconsistent indentation.